### ISSUE TITLE
Install error

### ISSUE THREAD

**Issue by Rytiss** [jdanthinne/advancedfeaturesvalues#1]
I'm getting an installation error: `Class FeatureValueOverrideOriginal54f9b22617779 does not exist.`

ID changes at random when retrying installation.

**Comment by jdanthinne** [comment]
What is your Prestashop version?

**Comment by Rytiss** [comment]
1.6.0.13 (I see there is 1.6.0.14 now...)

Pagarbiai,
Rytis Slatkevičius
+370 670 77777

**Comment by jdanthinne** [comment]
This is strange. I've developed the module using 1.6.0.13 as well.
How did you install the module? What is the name of the folder you put in /modules?

**Comment by Rytiss** [comment]
I used "advancedfeaturesvalues" as the name, tried to install through the
modules panel. All files were changed to 777 permissions (although in the
initial test I did not do that). I also tried the 777 on the whole
prestashop directory, did not work.

Manually copying the files and modding the DB worked (although I have one
bug, I'll make a pull request on Monday).

## 

Pagarbiai / Sincerely
Rytis Slatkevičius
+370 670 77777

**Comment by jdanthinne** [comment]
Can you try the new version I've just pushed. It includes a .zip archive for easy install, and clear the caches on (un)install. Still working fine for me on a fresh Prestashop install.

**Comment by jdanthinne** [comment]
I can see the bug now. I've just tried to install the module on another hosted website and getting the same error… Still searching for the cause (thought it was permissions, but 777 didn't change the error).

**Comment by jdanthinne** [comment]
Ok, it seems that this is working fine with PHP 5.5 but not 5.4. Trying to figure why.

**Comment by ValentinMilea** [comment]
Hi, Did you manage to figure out the PHP 5.4 issue? Thank you.

**Comment by jdanthinne** [comment]
No, not yet. If someone knows about 5.4 / 5.5 differences (in classes inheritance and overrides), he's welcome to have a look.

**Comment by daxpunk** [comment]
J'avais le même souci avec AdminFeaturesController, j'ai renommé le fichier en : _AdminFeaturesController.php et ça à fonctionné.

**Comment by jdanthinne** [comment]
I've just tried that, but no change…

**Comment by jdanthinne** [comment]
To clarify, what I mean by 'no change' is that the install process is ok now, but the renamed file is not installed as an override. So, pointless…

**Comment by jdanthinne** [comment]
Can you try the latest version? I've made some commits that seem to fix the installation. I've managed to install the module with PHP 5.2.17.

**Comment by ValentinMilea** [comment]
Hi,

I just installed the module succesfully. I will get back with more feedback after I play with it a little.

Thank you,
-Valentin
